test/oracles/centralized_oracle.js

```diff
@@ -38,6 +38,8 @@ contract('CentralizedOracle', function(accounts) {
         _resultSettingStartBlock: 70,
         _resultSettingEndBlock: 90
     };
+    const NUM_OF_RESULTS = 3;
+    const VERSION = 0;
```

Comment:
3 is an arbitrary number for testing right? Could you add comment to it?

Reply:
actually its not arbitrary. its meant to match the number of results in the defined TopicEvent params above it. adding note.